README.rst

```diff
@@ -78,6 +80,96 @@ printed instead.
                             is generated.
+                            
+The generate_stix tool is used for more general object generation, which does
+not require a language file to be specified:
+
+::
+
+    usage: generate_stix.py [-h] [--min-rels MIN_RELS] [--max-rels MAX_RELS]
+                        [--p-reuse P_REUSE] [--p-sighting P_SIGHTING]
+                        [--dangling-refs] [--ref-max-depth REF_MAX_DEPTH] [-v]
+                        [--stix-version {2.0,2.1}] [-b]
+
+    Generation random STIX content
+
+    optional arguments:
+      -h, --help            show this help message and exit
+      --min-rels MIN_RELS   Minimum number of SROs to create. Default=1
+      --max-rels MAX_RELS   Maximum number or SROs to create. Default=5
+      --p-reuse P_REUSE     Probability of object reuse, when creating new 
+                        connections among objects. Must be a real number in
+                        [0, 1]. Lower values result in a graph with more nodes
+                        and less interconnection. Higher values result in a
+                        graph with fewer nodes and more interconnection.
+                        Default=0.5
+  --p-sighting P_SIGHTING
+                        Probability that when an SRO is added, it is a
+                        sighting. Must be a real number in [0, 1]. Default=0.1
+  --dangling-refs       Leave reference properties "dangling". Don't force
+                        them to refer to existing objects. Applies to all
+                        reference properties *except* the endpoints of SROs.
+  --ref-max-depth REF_MAX_DEPTH
+                        If creating a new object to avoid a dangling reference,
+                        the new object could itself have reference properties;
+                        new objects created to satisfy those could themselves
+                        have reference properties, etc. This setting limits how
+                        far we grow this "reference graph". Enforcement of this
+                        limit is best-effort; reference properties required by
+                        the specification may cause further growth. Only
+                        applicable if --dangling-refs is not given. Must be a
+                        non-negative integer. Default=0
+  -v, --verbose         Enable verbose diagnostic output. Repeat for increased
+                        verbosity.
+  --stix-version {2.0,2.1}
+                        STIX version to use. Default=2.1
+  -b, --bundle          Create a bundle
+
+Python Library
+~~~~~~~~~~~~~~
+
+You can also generate STIX objects programmatically in a Python script. This can
+be useful when the generated objects are needed programmatically in Python code.
+
+The fastest and easiest way to create random STIX objects is with the
+create_stix_object function:
+
+.. code-block:: python
+
+    import stix2generator
+
+    generated = stix2generator.create_stix_object()
```

Comment:
Was this intended to demonstrate the STIXGenerator?  Was it intended to be:

```suggestion
    generator = stix2generator.create_stix_generator()
    generated = generator.generate()
```
?